### PR TITLE
Don't re-render changes row if we don't get an initialised changes struct, and make cache big enough for main-net

### LINKED ISSUE #225: Max Delta support seems broken
gettin results like this on mainnet, don't feel super high pri on solving this cause it's no one's favorite feature, but wanted to note it down 

Screen Shot 2022-07-13 at 3 20 01 PM

### LINKED ISSUE #239: program changes table sorting has flip-flops of rows if the sort function is identical
I think all we need is a secondary tie-breaker based on public-key.

### LINKED ISSUE #259: SOL amount likes to wink in and out on pinned accounts
SOL amount likes to wink in and out on faved accounts at the moment

Maybe related to #225?

Screen Shot 2022-07-06 at 3 03 36 PM
Screen Shot 2022-07-06 at 3 03 19 PM

_Originally posted by @nathanleclaire in https://github.com/workbenchapp/solana-workbench/issues/173#issuecomment-1176796135_

### PR DESCRIPTION
**Pull request by SvenDowideit**
- but mainnet shows there's more

closes #259
closes #225
closes #239

Signed-off-by: Sven Dowideit <SvenDowideit@home.org.au>

### CODE REVIEW

File: src/renderer/components/ProgramChange.tsx [hunk 1]
```diff
@@ -42,6 +42,10 @@ export function ProgramChange(props: {
   useEffect(updateAccount, [updateAccount]);
   useInterval(updateAccount, 666);
 
+  if (!change) {
```
Comment (SvenDowideit) [inline src/renderer/components/ProgramChange.tsx:45]: talk about maddening. this shouldn't work, but it does. 
its also a compile error, but it runs (if you make the change while already running.

**Comment by SvenDowideit** [comment]
okay! the flicking might be due to the default return from getAccounts when there's a cache miss - which looks like it happens alot on mainnet - presumably because there's so much going on.

File: src/renderer/components/ProgramChange.tsx (hunk 1 above)
Comment (SvenDowideit) [reply src/renderer/components/ProgramChange.tsx:49]: ah, `return null` was what i needed to know

**Comment by SvenDowideit** [comment]
this should properly go away when we do #230

**Comment by nathanleclaire** [comment]
This is great!! Nicely done! 🎉 

LGTM

**Pull request merged**